src/Misc.ino

```diff
@@ -2127,6 +2136,17 @@ void transformValue(
                 }
               }
               break;
+            case 'C': //Capitalize First Word-Character value (space/dash/period are checked)
+              if (value.length() > 0) {
+                value.toLowerCase();
+                bool nextCapital = true;
+                for (uint8_t i = 0; i < value.length();i++) {
+                  if (nextCapital)
+                    value[i] = toupper(value[i]);
+                  nextCapital = (value[i] == ' ' || value[i] == '-' || value[i] == '.'); // Very simple, capitalize-first-after-space/dash/period
```

Comment:
I don't think the `-` is a good separator trigger for detecting words.
Maybe not used that often in English, but in Dutch (and other languages), the - is used a lot in words.  (as you should know, judging your name :) )

Reply:
Agreed, removed it.